### PR TITLE
createrepo proposal (#1403)

### PR DESCRIPTION
**Pull request by keichwa**
https://github.com/SUSE/spacewalk/issues/16827
https://bugzilla.suse.com/show_bug.cgi?id=1195294
* more general description; per feedback
* Update modules/administration/pages/custom-channels.adoc
Co-authored-by: Michael Calmer <mc@suse.de>

# Description

Short summary of why you created this PR

# Target branches

Which documentation version does this PR apply to?

- [ ] Master (Default)
- [x] Manager-4.2
- [ ] Manager-4.1
- [ ] Manager-4.0

# Links

Fixes #<insert issue or PR link, if any>

### CODE REVIEW

File: .changelog [hunk 1]
```diff
@@ -4,6 +4,11 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- In Custom Channels chapter of the Administration Guide, provide information
+  about creating metadata (bsc#1195294)
+- In Client Configuration Guide, integrated SLE Micro Client documentation
+  next to SUSE Linux Enterprise Client documentation and other related
+  documentation improvements (bsc#1195145)
```
Comment (keichwa) [inline .changelog:11]:
```suggestion
```

**Comment by keichwa** [comment]
I hope I got it right.

**Review by jcayouette — APPROVED** [review]

**Review by jcayouette — CHANGES_REQUESTED** [review]
Looks good but I see some conflicts in the changelog.

File: .changelog [hunk 2]
```diff
@@ -4,15 +4,8 @@
...
-- Documented GPG encrypted Salt Pillars in the Salt book.
-- Clarified channel label name in Registering Clients with RHUI section of the
-  Client Configuration Guide (bsc#1196067)
-- In Throubleshooting Synchronization chapter in the Administration Guide
-  added instructions for GPG removal.
-- In Client Configuration Guide, integrated SLE Micro Client documentation
-  next to SUSE Linux Enterprise Client documentation and other related
-  documentation improvements (bsc#1195145)
+- In Custom Channels chapter of the Administration Guide, provide information
+  about creating metadata (bsc#1195294)
```
Comment (keichwa) [inline .changelog:8]:
```suggestion
  about creating metadata (bsc#1195294)
- In the Client Configuration Guide, mark Yomi as unsupported on SLES 11 and 12
  - Documented GPG encrypted Salt Pillars in the Salt book.
- Clarified channel label name in Registering Clients with RHUI section of the
  Client Configuration Guide (bsc#1196067)
- In Throubleshooting Synchronization chapter in the Administration Guide
  added instructions for GPG removal.
- In Client Configuration Guide, integrated SLE Micro Client documentation
  next to SUSE Linux Enterprise Client documentation and other related
  documentation improvements (bsc#1195145)
```

File: .changelog [hunk 3]
```diff
@@ -4,8 +4,10 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)#Changelog_section_.28.25changelog.29
 
+- In Custom Channels chapter of the Administration Guide, provide information
+  about creating metadata (bsc#1195294)
 - In the Client Configuration Guide, mark Yomi as unsupported on SLES 11 and 12
-- Documented GPG encrypted Salt Pillars in the Salt book.
+  - Documented GPG encrypted Salt Pillars in the Salt book.
```
Comment (keichwa) [inline .changelog:10]:
```suggestion
- Documented GPG encrypted Salt Pillars in the Salt book.
```

**Pull request merged**